src/main/resources/assets/tconstruct/book/encyclopedia/en_us/defense/special/simple_absorption_absorption_efficiency.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We will want book pages for puny melting/mighty smelting as well, since the modifier tags will add those automatically as blank pages otherwise.